### ISSUE TITLE
Fix Deprecation Warnings

### ISSUE THREAD

**Issue by tristanlatr** [LesFruitsDefendus/saskatoon-ng#69]
When running tests, it raises lots óf deprecation warnings like this one:

```
RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
```

I think a automated search/replace could fix the issues.

**Comment by qusvv** [comment]
I fixed the ugettext_lazy one in 367bccca9bea6ea7f108173f271691234efe3844
However there are still warnings about BooleanFields:
```
harvest.Property.authorized: (fields.W903) NullBooleanField is deprecated. Support for it (except in historical migrations) will be removed in Django 4.0.
	HINT: Use BooleanField(null=True) instead.
```

**Comment by qusvv** [comment]
Removed NullBooleanFields
See commit 38cb6e7b2d86573a9173205644f69b338a29ec63
Warning: new migration file was created in harvest/migrations/
